drivers/scheduler/dcos/dcos.go

```diff
@@ -475,6 +475,16 @@ func (d *dcos) RescanSpecs(specDir string) error {
 	return nil
 }
 
+func (d *dcos) EnableSchedulingOnNode(n node.Node) error {
+	// TODO implement this method
+	return nil
```

Comment:
instead of nil, can you change this to return a not supported error? Let's start doing that for all new interface methods that aren't supported by these drivers.

The pattern to use is something like this: https://github.com/libopenstorage/cloudops/blob/master/unsupported/unsupported.go

Have all drivers inherit an unsupported driver.

Reply:
This is not done

Reply:
done